rust/worker/src/bin/query_service.rs

```diff
@@ -2,5 +2,8 @@ use worker::query_service_entrypoint;
 
 #[tokio::main]
 async fn main() {
+    tracing_subscriber::fmt()
```

Comment:
Interesting, was this the fix for that issue? What is it by default.

@nicolasgere @beggers - how do you think we should handle the log level config?

Reply:
It is definitely less than info level since info level logs were not getting printed

Reply:
> how do you think we should handle the log level config?

Could you be more specific? Like, how should we configure it, how should we propagate it, how should we think about different logging levels, something else?

Reply:
How we configure it and how we use the levels